server/spinwick.go

```diff
@@ -226,8 +237,16 @@ func (s *Server) updateSpinWick(pr *model.PullRequest) (string, bool, error) {
 		return installationID, true, errors.Wrap(err, "encountered error waiting for installation to become stable")
 	}
 
+	// Remove old messate to reduce the amount of similar messages and avoid confusion
+	if errComments == nil {
```

Comment:
It looks like we should be running `s.getComments()` before this?

Reply:
we get the comments in the beginning which will be the same and also we dont use the request against github, we save the token :)

Reply:
Okay!